src/loader.js

```diff
@@ -66,11 +67,12 @@ export class Loader {
         programId: this.programId,
         userdata,
       });
-      await sendAndConfirmTransaction(this.connection, program, transaction);
+      transactions.push(await sendAndConfirmTransaction(this.connection, program, transaction));
```

Comment:
don't `await` here.   We want to simply collect all the `Promise`s into the transactions array here, not `await` for them to resolve.

Reply:
Good point!